### DISCUSSION TITLE
怎么刷HOT100

### DISCUSSION THREAD

**Discussion by congvee** [shuzijun/leetcode-editor#587] [General]
怎么把HOT100的题目筛选出来

**Comment by congvee** [comment]
我找到了

**Reply by 474914041** [reply]
怎么找？

**Reply by shuzijun** [reply]
<img width="509" alt="image" src="https://github.com/shuzijun/leetcode-editor/assets/27667179/484ee896-d0f8-4279-b404-3c2428589588">

**Reply by xiaobailail** [reply]
没有这个选项啊
